File: STRINGS/1581/main.dart
import 'dart:io';

String language(int k) {
  Set<String> languages = {};
  while (k-- > 0) languages.add(stdin.readLineSync()!);
  return languages.length == 1 ? languages.first : 'ingles';
}

void main() {
  int n = int.parse(stdin.readLineSync()!);
  while (n-- > 0) {
    int k = int.parse(stdin.readLineSync()!);
    print(language(k));
  }
}

File: STRINGS/1581/main.rb
# frozen_string_literal:true

require 'set'

def language(k)
  languages = Set[]
  k.times { languages.add(gets) }
  languages.length == 1 ? languages.first : 'ingles'
end

n = gets.to_i
n.times do
  k = gets.to_i
  puts language(k)
end
